docker_28: 28.2.2 -> 28.3.2, docker_25: 25.0.10 -> 25.0.11

teutat3s opened the pull request:
Upstream changelogs:
`25.x`
https://github.com/moby/moby/compare/v25.0.10...v25.0.11
Upstream did not tag a new release for `docker/cli`, and there are no new
commits in the 25.0 branch:
https://github.com/docker/cli/compare/43987fca488a535d810c429f75743d8c7b63bf4f...25.0

`28.x`
https://github.com/moby/moby/compare/v28.2.2...v28.3.1
https://github.com/docker/cli/compare/v28.2.2...v28.3.1

Milestone 25.0.11
https://github.com/moby/moby/milestone/197?closed=1

Milestone 28.3.0
https://github.com/moby/moby/milestone/200?closed=1
https://github.com/docker/cli/milestone/147?closed=1

Milestone 28.3.1
https://github.com/moby/moby/milestone/204?closed=1
No commits here:
https://github.com/docker/cli/milestone/148?closed=1

Milestone 28.3.2
https://github.com/moby/moby/milestone/205?closed=1
https://github.com/docker/cli/milestone/149?closed=1

- [x] Checked that there were no changes to https://github.com/moby/moby/tree/v28.3.2/hack/dockerfile/install/.
- [x] Checked that there were no changes to https://github.com/moby/moby/tree/v25.0.11/hack/dockerfile/install/.

<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

Closes https://github.com/NixOS/nixpkgs/pull/419867.

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [Nixpkgs 25.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/doc/release-notes/rl-2511.section.md) (or backporting [25.05](https://github.com/NixOS/nixpkgs/blob/master/doc/manual/release-notes/rl-2505.section.md) Nixpkgs Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
- [NixOS 25.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2511.section.md) (or backporting [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) NixOS Release notes)
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md), [pkgs/README.md](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md), [maintainers/README.md](https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md) and other contributing documentation in corresponding paths.

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

teutat3s committed:
docker: 28.2.2 -> 28.3.2

https://github.com/moby/moby/compare/v28.2.2...v28.3.2
https://github.com/docker/cli/compare/v28.2.2...v28.3.2

kashw2 commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 422426`
Commit: `2e96b3083007f5eedaf969d1ba1456fc36a33926`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 19 packages built:</summary>
  <ul>
    <li>airlift</li>
    <li>airlift.dist</li>
    <li>charliecloud</li>
    <li>devcontainer</li>
    <li>docker (docker_28)</li>
    <li>docker-client</li>
    <li>docker-gc</li>
    <li>docker-language-server</li>
    <li>docker-sbom</li>
    <li>docker-vackup</li>
    <li>fn-cli</li>
    <li>pipework</li>
    <li>python312Packages.jupyter-repo2docker</li>
    <li>python312Packages.jupyter-repo2docker.dist</li>
    <li>python313Packages.jupyter-repo2docker</li>
    <li>python313Packages.jupyter-repo2docker.dist</li>
    <li>tarsum</li>
    <li>tests.devShellTools.nixos</li>
    <li>wrkflw</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 19 packages built:</summary>
  <ul>
    <li>airlift</li>
    <li>airlift.dist</li>
    <li>charliecloud</li>
    <li>devcontainer</li>
    <li>docker (docker_28)</li>
    <li>docker-client</li>
    <li>docker-gc</li>
    <li>docker-language-server</li>
    <li>docker-sbom</li>
    <li>docker-vackup</li>
    <li>fn-cli</li>
    <li>pipework</li>
    <li>python312Packages.jupyter-repo2docker</li>
    <li>python312Packages.jupyter-repo2docker.dist</li>
    <li>python313Packages.jupyter-repo2docker</li>
    <li>python313Packages.jupyter-repo2docker.dist</li>
    <li>tarsum</li>
    <li>tests.devShellTools.nixos</li>
    <li>wrkflw</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:white_check_mark: 15 packages built:</summary>
  <ul>
    <li>airlift</li>
    <li>airlift.dist</li>
    <li>devcontainer</li>
    <li>docker (docker-client, docker_28)</li>
    <li>docker-gc</li>
    <li>docker-language-server</li>
    <li>docker-sbom</li>
    <li>docker-vackup</li>
    <li>fn-cli</li>
    <li>python312Packages.jupyter-repo2docker</li>
    <li>python312Packages.jupyter-repo2docker.dist</li>
    <li>python313Packages.jupyter-repo2docker</li>
    <li>python313Packages.jupyter-repo2docker.dist</li>
    <li>tarsum</li>
    <li>wrkflw</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 15 packages built:</summary>
  <ul>
    <li>airlift</li>
    <li>airlift.dist</li>
    <li>devcontainer</li>
    <li>docker (docker-client, docker_28)</li>
    <li>docker-gc</li>
    <li>docker-language-server</li>
    <li>docker-sbom</li>
    <li>docker-vackup</li>
    <li>fn-cli</li>
    <li>python312Packages.jupyter-repo2docker</li>
    <li>python312Packages.jupyter-repo2docker.dist</li>
    <li>python313Packages.jupyter-repo2docker</li>
    <li>python313Packages.jupyter-repo2docker.dist</li>
    <li>tarsum</li>
    <li>wrkflw</li>
  </ul>
</details>

MisileLab commented:
https://github.com/docker/cli/releases/tag/v28.3.1 released

MisileLab commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 422426`

Logs: https://github.com/MisileLab/nixpkgs-review-gha/actions/runs/16128470584


---
### `x86_64-linux` (sandbox = true)
<details>
  <summary>:white_check_mark: 19 packages built:</summary>
  <ul>
    <li>airlift</li>
    <li>airlift.dist</li>
    <li>charliecloud</li>
    <li>devcontainer</li>
    <li>docker (docker_28)</li>
    <li>docker-client</li>
    <li>docker-gc</li>
    <li>docker-language-server</li>
    <li>docker-sbom</li>
    <li>docker-vackup</li>
    <li>fn-cli</li>
    <li>pipework</li>
    <li>python312Packages.jupyter-repo2docker</li>
    <li>python312Packages.jupyter-repo2docker.dist</li>
    <li>python313Packages.jupyter-repo2docker</li>
    <li>python313Packages.jupyter-repo2docker.dist</li>
    <li>tarsum</li>
    <li>tests.devShellTools.nixos</li>
    <li>wrkflw</li>
  </ul>
</details>


---
### `aarch64-linux` (sandbox = true)
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>devcontainer</li>
    <li>docker-language-server</li>
    <li>tests.devShellTools.nixos</li>
    <li>wrkflw</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 15 packages built:</summary>
  <ul>
    <li>airlift</li>
    <li>airlift.dist</li>
    <li>charliecloud</li>
    <li>docker (docker_28)</li>
    <li>docker-client</li>
    <li>docker-gc</li>
    <li>docker-sbom</li>
    <li>docker-vackup</li>
    <li>fn-cli</li>
    <li>pipework</li>
    <li>python312Packages.jupyter-repo2docker</li>
    <li>python312Packages.jupyter-repo2docker.dist</li>
    <li>python313Packages.jupyter-repo2docker</li>
    <li>python313Packages.jupyter-repo2docker.dist</li>
    <li>tarsum</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>devcontainer</summary>
<pre>
Done in 79.45s.
yarn run v1.22.22
$ node esbuild.js --production
Done in 1.35s.
buildPhase completed in 1 minutes 22 seconds
Running phase: installPhase
@nix { "action": "setPhase", "phase": "installPhase" }
Running phase: fixupPhase
@nix { "action": "setPhase", "phase": "fixupPhase" }
shrinking RPATHs of ELF executables and libraries in /nix/store/f4hmzqngh8z0i3p2mzscpdm75khwrj8i-devcontainer-0.77.0
shrinking /nix/store/f4hmzqngh8z0i3p2mzscpdm75khwrj8i-devcontainer-0.77.0/bin/devcontainer
shrinking /nix/store/f4hmzqngh8z0i3p2mzscpdm75khwrj8i-devcontainer-0.77.0/libexec/node_modules/@esbuild/linux-arm64/bin/esbuild
patchelf: cannot find section '.dynamic'. The input file is most likely statically linked
shrinking /nix/store/f4hmzqngh8z0i3p2mzscpdm75khwrj8i-devcontainer-0.77.0/libexec/node_modules/node-pty/build/Release/pty.node
checking for references to /build/ in /nix/store/f4hmzqngh8z0i3p2mzscpdm75khwrj8i-devcontainer-0.77.0...
patchelf: cannot find section '.dynamic'. The input file is most likely statically linked
patching script interpreter paths in /nix/store/f4hmzqngh8z0i3p2mzscpdm75khwrj8i-devcontainer-0.77.0
/nix/store/f4hmzqngh8z0i3p2mzscpdm75khwrj8i-devcontainer-0.77.0/libexec/scripts/gitAskPass.sh: interpreter directive changed from "#!/bin/sh" to "/nix/store/840zd1vd2kqwsraq0xfyrsrnrvsancjf-bash-5.2p37/bin/sh"
stripping (with command strip and flags -S -p) in  /nix/store/f4hmzqngh8z0i3p2mzscpdm75khwrj8i-devcontainer-0.77.0/libexec /nix/store/f4hmzqngh8z0i3p2mzscpdm75khwrj8i-devcontainer-0.77.0/bin</pre>
</details>
<details>
<summary>docker-language-server</summary>
<pre>∅  internal/cache
✓  internal/bake/hcl/parser (6ms)
✓  internal/bake/hcl (154ms)
∅  internal/configuration
∅  internal/pkg/cli
∅  internal/pkg/cli/metadata
✓  internal/compose (119ms)
∅  internal/pkg/lsp/textdocument
∅  internal/pkg/server
∅  internal/tliron/glsp
∅  internal/tliron/glsp/protocol
∅  internal/tliron/glsp/server
✓  internal/telemetry (4ms)
✓  internal/pkg/document (34ms)
✓  internal/types (10ms)

=== Skipped
=== SKIP: internal/bake/hcl TestLocalDockerfileForWindows (0.00s)

DONE 1305 tests, 1 skipped in 5.593s</pre>
</details>
<details>
<summary>wrkflw</summary>
<pre>   Compiling equivalent v1.0.2
   Compiling http v0.2.12
   Compiling tracing-core v0.1.33
   Compiling httparse v1.10.1
   Compiling utf16_iter v1.0.5
   Compiling tracing v0.1.41
   Compiling utf8_iter v1.0.4
   Compiling try-lock v0.2.5
   Compiling foreign-types-shared v0.1.1
   Compiling openssl v0.10.72
   Compiling write16 v1.0.0
   Compiling foreign-types v0.3.2
   Compiling want v0.3.1
   Compiling http-body v0.4.6
   Compiling num-integer v0.1.46
   Compiling zerofrom-derive v0.1.6
   Compiling yoke-derive v0.7.5
   Compiling serde_derive v1.0.219
   Compiling zerovec-derive v0.10.3
   Compiling displaydoc v0.2.5</pre>
</details>
</details>


---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:white_check_mark: 15 packages built:</summary>
  <ul>
    <li>airlift</li>
    <li>airlift.dist</li>
    <li>devcontainer</li>
    <li>docker (docker-client, docker_28)</li>
    <li>docker-gc</li>
    <li>docker-language-server</li>
    <li>docker-sbom</li>
    <li>docker-vackup</li>
    <li>fn-cli</li>
    <li>python312Packages.jupyter-repo2docker</li>
    <li>python312Packages.jupyter-repo2docker.dist</li>
    <li>python313Packages.jupyter-repo2docker</li>
    <li>python313Packages.jupyter-repo2docker.dist</li>
    <li>tarsum</li>
    <li>wrkflw</li>
  </ul>
</details>


---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:white_check_mark: 15 packages built:</summary>
  <ul>
    <li>airlift</li>
    <li>airlift.dist</li>
    <li>devcontainer</li>
    <li>docker (docker-client, docker_28)</li>
    <li>docker-gc</li>
    <li>docker-language-server</li>
    <li>docker-sbom</li>
    <li>docker-vackup</li>
    <li>fn-cli</li>
    <li>python312Packages.jupyter-repo2docker</li>
    <li>python312Packages.jupyter-repo2docker.dist</li>
    <li>python313Packages.jupyter-repo2docker</li>
    <li>python313Packages.jupyter-repo2docker.dist</li>
    <li>tarsum</li>
    <li>wrkflw</li>
  </ul>
</details>

MisileLab commented:
https://github.com/docker/cli/releases/tag/v28.3.2 released

drupol commented:
Please upgrade to the very latest version and I'll merge this.

teutat3s committed:
docker_25: 25.0.10 -> 25.0.11

https://github.com/moby/moby/milestone/197?closed=1

Upstream did not tag a new release for docker/cli, and there are no new
commits in the 25.0 branch:
https://github.com/docker/cli/compare/43987fca488a535d810c429f75743d8c7b63bf4f...25.0

https://github.com/moby/moby/compare/v25.0.10...v25.0.11
